### ISSUE TITLE
Request for improvement from Cote d'Ivoire NMS

### ISSUE THREAD

**Issue by mhabimana** [climsoft/Climsoft#652]
I had a virtual meeting with @berenger07 from Cote d'Ivoire NMS yesterday,17/05/2021 and was pleased to see that they managed to install and customised Climsoft version4 to suit their needs on their own following online resource material on Climsoft. Following areas of improvement on the software were requested:
1. Translation of the software in French to have dialogues in French;
2. Enhanced the dialog for observations updates to include drop down text box on StationID and ElementID to ease the task of selection of both station and element instead of typing these;
3. Include "Tri-Hourly" products in the list of Climsoft products beside the Hourly products.
@berenger07,not sure I missed any thing,please feel free to add what I may left out.

**Comment by rdstern** [comment]
@mhabimana this is very timely as we have our meeting together with Karen McCourt on Thursday to plan the activities for this year.
So, this is also a good time to look through the issues for Climsoft and for @berenger07 and others to make any more suggestions for our consideration.

I can also comment that we have been struggling mightily to include the first item (French version) in R-Instat.  It is taking a long time, but we are convinced that the method is ok.  And the experience should hopefully be much simpler the second-time round, i.e. in Climsoft.  So I would agree that the French translation should be high on the priority list.

The second item - of improving the observations dialogue - sounds very sensible and relatively small.
The third item also sounds very useful.  I am keen to know what specific products would be useful for the 3-hour data.  (I would also like to offer to produce similar products using R-Instat, but we don't yet have good ways of analysing within-day data.  That's on my list of improvements I would like to see.)

**Comment by berenger07** [comment]
yes I think it's all there. Thank you for this zoom meeting which allowed me to remove several doubts. And concerning the French version it would be welcome. Even if with this one we manage to make our entries correctly. The areas for improvement proposed once integrated, we will make it fully operational on Climsoft.
Thank you

KOUACOU Adou Kouassi Doré BerengerIngénieur en Télédétection, SIG et Bases de Données Direction de la météorologie Nationale (SODEXAM)Contact: +225 0708736793/+225 0505655191Boîte Postale: 10 BP 3310 Abidjan 10

 

    Le mardi 18 mai 2021, 17:00:51 UTC+1, rdstern ***@***.***> a écrit :  
 
 


@mhabimana this is very timely as we have our meeting together with Karen McCourt on Thursday to plan the activities for this year.
So, this is also a good time to look through the issues for Climsoft and for @berenger07 and others to make any more suggestions for our consideration.

I can also comment that we have been struggling mightily to include the first item (French version) in R-Instat. It is taking a long time, but we are convinced that the method is ok. And the experience should hopefully be much simpler the second-time round, i.e. in Climsoft. So I would agree that the French translation should be high on the priority list.

The second item - of improving the observations dialogue - sounds very sensible and relatively small.
The third item also sounds very useful. I am keen to know what specific products would be useful for the 3-hour data. (I would also like to offer to produce similar products using R-Instat, but we don't yet have good ways of analysing within-day data. That's on my list of improvements I would like to see.)
